lib/models/Session.js

```diff
@@ -17,12 +15,6 @@ class Session {
     this.button = device
     this.location = device
   }
-
-  // this is specific to buttons: notice that this.alertType is updated
-  incrementButtonPresses(numberOfAlerts) {
-    this.numberOfAlerts += numberOfAlerts
-    this.alertType = ALERT_TYPE.BUTTONS_URGENT
-  }
```

Comment:
This change will require some testing. We'll need to make sure that 2+ button presses are still categorized as Urgent.

It looks like the change is in place to make this happen (https://github.com/bravetechnologycoop/BraveButtons/blob/c65b1dd17ab762b7c48cc1f1d615ac0cf4a05a2b/server/buttonAlerts.js#L45), but it would be worth double-checking.

Reply:
Ah, yes, good catch! I'll add a test case for that in the next to-be-merged PR for BraveButtons.